app-v2/src/utils/metrics.ts

```diff
@@ -0,0 +1,19 @@
+import { COMBAT_SKILLS, MEMBER_SKILLS, Player } from "@wise-old-man/utils";
+
+export function getBuildContextMetrics(player: Player, metrics: any[]) {
```

Comment:
Instead of using it like
`const filteredRows = getBuildContextMetrics(player, rows)`, you can use it like:

```
const visibleMetrics = getBuildContextMetrics(player); // where it returns Metric[]
const filteredRows = rows.filter(row => visibleMetrics.includes(row.metric));
```

this should make it a little more flexible to use in the future, and not completely dependent on the rows having a `metric` field inside them.

PS: just noticed, you don't need to pass `player: Player` in the first argument, just `build: PlayerBuild`